providers/flagd/src/test/java/dev/openfeature/contrib/providers/flagd/resolver/process/storage/FlagStoreTest.java

```diff
@@ -98,9 +99,9 @@ public void changedFlags() throws Exception {
         Map<String, FeatureFlag> expectedChangedFlags =
                 FlagParser.parseString(getFlagsFromResource(VALID_LONG),true);
```

Comment:
This call would intermittently fail in Java 9+ due to concurrent modification exceptions.